pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>de.zalando.ep</groupId>
     <artifactId>zalenium</artifactId>
-    <version>3.141.59z</version>
```

Comment:
Not sure how the pom version managed (by manual or by script). Let me know if I need to change it or leave it unchanged.

Reply:
This won't be changed as no release will be done. I am happy to run the CI to test the PR and merge it, but this will be left unchanged.